src/server.js

```diff
@@ -248,8 +243,7 @@ function init(socket, client) {
 								const obj = {};
 
 								if (success) {
-									obj.success = "Successfully updated your password, all your other sessions were logged out";
-									obj.token = client.config.token;
+									obj.success = "Successfully updated your password";
```

Comment:
Honestly, because we have that already, yes. I am happy to not do that anymore when #986 arrives, i.e. when there is an alternate way to do this, but I very strongly welcomed you allowing this in the first place in #370.

So, sure, it's not perfect, but it's times better than not having any option for this. Also, I'm not too worried about push notifications, because if the behavior is meant to kill sessions everywhere, there is no reason to keep receiving notifications.

Again, more than happy to change my mind when #986 makes it (and it's great stuff, I really want that in!) but "not perfect" vs. "back to nothing" is an easy choice to me :)